README.md

```diff
@@ -152,7 +152,7 @@ babili     21.9kB  219%    7.46kB 164%     2ms        722ms
 closure js 24.01kB 191%    8.04kB 144%     2ms        3301ms
 ```
 
-Run with: `./scripts/benchmark.js ./scripts/fixtures/backbone.js`
+Run with: `./scripts/benchmark.js backbone` **or** `./scripts/benchmark.js backbone backbone/backbone.js`
```

Comment:
I don't think there's any need to specify the relative paths in these cases but you should probably specify the appropriate version so that it tallies up with the results table.